.github/workflows/build-test-application.yml

```diff
@@ -82,8 +84,7 @@ jobs:
       OIDC_CLIENT_SCOPES: openid
       SESSION_SECRET: secret
       SESSION_EXPIRY: 3600000
-      REDIS_HOST: localhost
-      REDIS_PORT: 6379
+      SESSION_STORE_TABLE_NAME: account-mgmt-frontend-SessionStore
```

Comment:
Will it help to add this env variable to the sample file ?

Reply:
done